packages/build/commands/build.js

```diff
@@ -42,5 +42,10 @@ module.exports = function defineBuildCommand(args) {
 };
 
 if (require.main === module) {
-  require('hops-local-cli').run(module.exports, 'build');
+  try {
+    require.resolve('hops-local-cli');
```

Comment:
Would it not make more sense to try using `hops` first and only fall back to the legacy version if needed? Applies to all occurences.